src/main/java/org/mule/tools/apikit/model/MuleConfigBuilder.java

```diff
@@ -52,10 +54,28 @@ public static MuleConfig fromDoc(Document muleConfigContent) {
         }
       }
     }
+    return new MuleConfig(httpListenerConfigs, apikitConfigs, flowsInConfig, muleConfigContent);
+  }
 
+
+  public static MuleConfig fromDocWithoutFlows(Document muleConfigContent) {
```

Comment:
This seems to a copy of the method above. Try to generalize the logic in one method only.